models/User.model.js

```diff
@@ -2,6 +2,9 @@ var mongoose = require('mongoose');
 var ObjectId = require('mongoose').Schema.Types.ObjectId;
 
 var OwnedGarmentSchema = new mongoose.Schema({
+    garment_id: ObjectId,
```

Comment:
Why garment_id versus using the mongo schema _id field?

Reply:
If the garment is deleted from the database, it should also be deleted from all of the users who have it. It is easier to do that if you delete by id. If duplicate garments are in the database, deleting by image link would delete everything that shares the link

Reply:
But all mongo schema objects have an _id property by default. This line doesn't need to be here, as _id can be used.